raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -75,6 +75,7 @@ library ChannelManagerLibrary {
             // do nothing, continue to create a new channel
         } else if (!contractExists(self, channel_address)) {
             deleteChannel(self, caller_address, partner, channel_address, caller_index, partner_index);
+            ChannelDelete(caller_address, partner, channel_address);
```

Comment:
Well could be in the `deleteChannel()` function. left it out because I did not if-else decision making ladders there and this specific `else if` was deleting the channel.